prow/config/config.yaml

```diff
@@ -730,22 +644,6 @@ presubmits:
           value: "TRUE"
         image: docker.io/golang:1.17
         imagePullPolicy: Always
-  - name: gofmt
-    branches:
-    - release-0.1
```

Comment:
what is this release-0.1, is it some legacy stuff  ?

Reply:
Thats IPAM